preprocessing.py

```diff
@@ -148,4 +151,7 @@ def process_file(input_file, output_file, use_gpu):
 
 def filter_input_files(input_files):
     disallowed_file_endings = (".gitignore", ".DS_Store")
-    return list(filter(lambda x: not x.endswith(disallowed_file_endings), input_files))
\ No newline at end of file
+    return list(filter(lambda x: not x.endswith(disallowed_file_endings), input_files))
+
+use_gpu=False
+process_raw_data(use_gpu, force_pre_processing_overwrite=True)
```

Comment:
Would prefer this to be in a separate file (maybe create preprocessing_cli.py?). Otherwise any library that wants to include just a part of preprocessing.py might accidentally run process_raw_data. If you create a cli you can also set use_gpu according to a user specified command line argument.

Reply:
Sounds good, I tried to implement this. I haven't used argparse before so you'll probably want to make some edits.